scripts/google/covid_harvard/StatisticalVariable.mcf

```diff
@@ -0,0 +1,27 @@
+Node: dcid:HarvardCOVID19CumulativeCases
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalCondition
+statType: dcs:measuredValue
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:ConfirmedCase
```

Comment:
Since we're now flushing out the schema, let's add
medicalCondition: dcs:COVID_19